### DISCUSSION TITLE
win11不能更新GFWlist

### DISCUSSION THREAD

**Discussion by RXUARJ** [v2rayA/v2rayA#1400] [Q&A]
显示
![image](https://github.com/v2rayA/v2rayA/assets/32945597/bd414390-1301-4464-94f0-ac1e7fd4b0f0)

Get "https://raw.githubusercontent.com/v2rayA/dist-v2ray-rules-dat/202404162209/geosite.dat": dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found.

**Comment by xiaozixun** [comment]
https://t.me/v2raya_zh/143067
现在是加host来解决，不过我这里只是偶尔才能成功。

**Reply by AnitaKamani** [reply]
Hi, I saw the Telegram message, but not sure where to update the configs. Can you point me to the right file name?

**Reply by irort** [reply]
> Hi, I saw the Telegram message, but not sure where to update the configs. Can you point me to the right file name?

Windows:   C:\Windows\System32\drivers\etc\hosts

**Reply by AnitaKamani** [reply]
found it thanks.
